src/app/hearing-part/components/draggable-hearing-part/draggable-hearing-part.component.ts

```diff
@@ -0,0 +1,26 @@
+import { AfterViewChecked, ChangeDetectionStrategy, Component, Input} from '@angular/core';
+
+import * as $ from 'jquery';
+import 'jquery-ui/ui/widgets/draggable.js';
+import { HearingPart } from '../../models/hearing-part';
+@Component({
+  selector: 'app-draggable-hearing-part',
+  templateUrl: './draggable-hearing-part.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DraggableHearingPartComponent implements AfterViewChecked {
+    @Input() hearingPart: HearingPart;
+
+    constructor() {
+    }
+
+    ngAfterViewChecked() {
+        ($('.draggable-hearing') as any).draggable({
+            revert: false,
+            helper: function () { return $(this).clone()
```

Comment:
plz double check this line. I'm pretty sure that `this` there will match function context, not a component class.
If you would like to match component class just change it to `() => { ... `

you can also change formatting, put `{` and `}` in new lines

Reply:
You are 100% percent right, the 'this' will match the function context which in 'helper' function is the currently dragged element and it is what I want it to be.